src/main/java/twilightforest/world/TFGenCaveStalactite.java

```diff
@@ -180,12 +289,16 @@ public boolean makeSpike(World world, Random random, int x, int y, int z, int ma
                 }
 
                 for (int dy = 0; dy != (spikeLength * dir); dy += dir) {
-                    setBlock(world, x + dx, y + dy, z + dz, blockID);
+                    setBlock(world, x + dx, y + dy, z + dz);
                 }
             }
         }
 
         return true;
     }
 
+    public void setBlock(World world, int x, int y, int z) {
+        setBlockAndMetadata(world, x, y, z, blockID, blockMeta);
+    }
+
```

Comment:
Instead of adding a new method, you can just override setBlockAndMetadata in the subclass, like so:
```java
    @Override
    protected void setBlockAndMetadata(World world, int x, int y, int z, Block block, int meta) {
        //  ...
    }
```

Reply:
Oh, that's the parent class, now I see